packages/pages/src/common/src/template/types.ts

```diff
@@ -52,7 +52,7 @@ export type TransformProps<T extends TemplateProps> = (props: T) => Promise<T>;
  *
  * @public
  */
-export type GetPath<T extends TemplateProps | void> = (props: T) => string;
+export type GetPath<T extends TemplateProps> = (props: T) => string;
```

Comment:
This shouldn't take in any props. You should also use it as the type for getPath which you changed to ()=>string

Reply:
This file is now reverted to main, which does take props but it's for the feature templates